src/Resources/views/admin/order/update/_order_items.html.twig

```diff
@@ -28,7 +28,7 @@
                 {% set orderItem = itemForm.vars.data %}
                 <tr>
                     <td>{{ form_row(itemForm.quantity) }}</td>
-                    <td>{{ orderItem.productName }}</td> {# todo add variant #}
+                    <td>{{ form_widget(itemForm.variant) }}</td>
```

Comment:
Why did you change this to `form_widget`. Is it so that _existing_ order lines can be edited? For 'safety reasons' I wouldn't do that myself. What do you think?

Reply:
I was 100% sure we want to do that :dance: I understand we do not :) 
Anyway, we should probably have a disabled field with variant here, as in other case the form nulls the value on the order item.

Reply:
Yes, I think I also might have said that :D That's also why I am asking a bit defensively :D 

> Anyway, we should probably have a disabled field with variant here, as in other case the form nulls the value on the order item.

Are you sure they are nulled when they are not posted? I don't think so, and it worked before where this widget wasn't there

Reply:
![Zrzut ekranu 2024-06-14 o 12 01 27](https://github.com/Setono/sylius-order-edit-plugin/assets/6212718/697dc306-33e3-465c-9cdd-b75a92589e70)

I've checked it one more time, and the variant is nulled :dancer: but we can switch to `PATCH` method and then it should be ok, wdyt?